### ISSUE TITLE
⚠️ Remove the Warnings and Console Logs from dev

### ISSUE THREAD

**Issue by itsomkathe** [navgurukul/bhanwari-devi#714]
The `dev` branch currently has many warnings and `console.log` statements. This is the case with both the `CMD` and `Dev Tools`. 
It gets really annoying when you want to log something on your branch but cannot find it. Also, it's not a good dev practice, though logs are removed in production. 

### ❌ DO NOT REMOVE THE LOGS FROM `logger.js` and `browser-matrix.min.js`

Most of the warnings are due to `key props`, value never used, etc.

**Things we need to fix:**

1.  ❗ Red Warnings
2. ⚠️ Yellow Warnings (Some cannot be fixed, so do as much you can)
3. 🏳️ `console.log` statements

### Branch `Warning-Logs` linked here should be used. Take a pull from `dev` every time.

**Things to remember:**

1. Do not break any feature 
2. Avoid changing files that are being changed or recently changed. Merge conflicts can happen
3. In case of merge conflicts, receive the incoming changes and fix them later **OR** contact the appropriate person on slack.

**Images for reference:**

![Capture](https://user-images.githubusercontent.com/34516916/191676746-a25cad77-478f-4f12-9911-2694c97faddf.PNG)

![Capture1](https://user-images.githubusercontent.com/34516916/191676832-d579dbdd-0fb8-4474-9948-9466a4aba7ce.PNG)

**Comment by yamitrvg12** [comment]
Hey [itsomkathe](https://github.com/itsomkathe), can you assign me this issue so that I can work on it?

**Comment by kartiks26** [comment]
> Hey [itsomkathe](https://github.com/itsomkathe), can you assign me this issue so that I can work on it?

Yes sure you can work on it and check there the branch already worked on @itsomkathe

**Comment by itsomkathe** [comment]
> Hey [itsomkathe](https://github.com/itsomkathe), can you assign me this issue so that I can work on it?

@yamitrvg12 I've assigned you. Go ahead!

**Comment by yamitrvg12** [comment]
working on it...

**Comment by yamitrvg12** [comment]
Hi [itsomkathe](https://github.com/itsomkathe)

I have 3 question:
1. What information should be on `.env.development`?
2. Is login works?
3. Should be working on Warning-Logs branch? because that branch is behind of 5 commit from main branch

**Comment by itsomkathe** [comment]
@yamitrvg12 

1. @kartiks26 have you given the credential details?
2. 
3. You can create a new branch from `Warning-Logs` as a base branch. Then raise a PR against `Warning-Logs` once you are done.

**Comment by itsomkathe** [comment]
@yamitrvg12 

You can refer to this Markdown 
https://github.com/navgurukul/bhanwari-devi/blob/contributing.md/CONTRIBUTING.md
